pom.xml

```diff
@@ -4896,6 +4896,18 @@
         <scope>test</scope>
       </dependency>
 
+      <!-- Confluent -->
+      <dependency>
+        <groupId>io.confluent</groupId>
+        <artifactId>kafka-avro-serializer</artifactId>
+        <version>5.0.0</version>
+      </dependency>
+      <dependency>
+        <groupId>io.confluent</groupId>
+        <artifactId>kafka-schema-registry-client</artifactId>
+        <version>5.0.0</version>
```

Comment:
Does it worth to extract a property for `kafka-avro-serializer` and `kafka-schema-registry-client` ?

Reply:
okay